demo/src/client/faro/initialize.ts

```diff
@@ -33,11 +33,10 @@ export function initializeFaro(): Faro {
         },
       }),
     ],
-    batching: {
-      // Batching is enabled by default and there is normally no reason to disable it.
-      // We did it in the demo so users can inspect each single requests sent due to certain interactions.
-      enabled: false,
-    },
+    // batching: {
```

Comment:
why is this commented out?

Reply:
Batching is enabled by default.
The example in the demo is made with one request per signal in mind.
I left it there as commented out code to peeps can disable batching to see a single request per signal.

But when thinking twice it makes no sense because if batching is disabled we will get a bunch of errors because the Faro fetch buffers will fill to quick (too many event in a very short amount of time).

Let me remove it